sql/1.0xx/activity_feed.sql

```diff
@@ -0,0 +1,13 @@
+BEGIN;
+
+CREATE TABLE "activity_feed" (
+  "id" serial NOT NULL,
+  "created" timestamp with time zone NOT NULL,
+  "type" text NOT NULL,
```

Comment:
That might work better - I was thinking of places this `type` filtering broke down and it's pretty easy to find them. Let's say we focus on areas of interest, e.g. "I want to know when any IA related to Pokemon has status changed to 'live'." it breaks down pretty quickly but this is inarguably a useful kind of filter.

The string manipulation also smells, I agree completely.

I was also considering a secondary meta-stash which is a postgres array of strings (again, free-form) so the subscription could look something like:
```perl
{
    type => 'iapage%updated',
    meta => [ 'status:live', 'category:pokemon', ],
}
```

I think having *n* tags is ultimately simpler and just as flexible. Applications can designate the hierarchy themselves, `tag1` is their top-level category, `tag2` is action, `tag3` name, `tag4` what's new, tag5 categories...

```perl
{
    tag1 => 'ia',
    tag2 => 'update',
    tag3 => 'pokemon',
    tag4 => 'status:live',
    tag5 => 'reference,geek,special-interest'
}
```

There's still a bit of string-y stuff in there, but 'status:live' can be an exact string match (it's an invented convention, but not as smelly as hierarchical-wildcarding :smile:).

Hmmm, pokemon isn't even an IA category - pretend I contrived a real world example *cough*. Going to give it some thought.

Thanks!